ooni/tests/mocks.py

```diff
@@ -242,3 +242,20 @@ def isSupported(self):
 
     def isReachable(self):
         return defer.succeed(True)
+
+
+class MockDnsResponsePayload(object):
+    def dottedQuad(self):
+        return '127.0.0.1'
+
+
+class MockDnsResponse(object):
+    payload = MockDnsResponsePayload()
+
+
+@defer.inlineCallbacks
+def mock_dns_lookup_address():
```

Comment:
Ah, cool, great to know! I was also a bit confused by this when helping @xrmx to review this.